pkg/minikube/registry/drvs/qemu2/qemu2.go

```diff
@@ -117,7 +117,8 @@ func qemuVersion() (semver.Version, error) {
 	if err != nil {
 		return semver.Version{}, err
 	}
-	v := strings.Split(strings.TrimPrefix(string(rr), "QEMU emulator version "), "\n")[0]
+	v := strings.Fields(strings.Split(strings.TrimPrefix(string(rr), "QEMU emulator version "), "\n")[0])[0]
```

Comment:
Thank you for this PR and making minikube better, how about make this as a failover if the first one didnt work try this one ? I am worried off unneeded nill pointer exceptions